board/batocera/fsoverlay/etc/init.d/S99userservices

```diff
@@ -17,28 +17,38 @@ then
 fi
 
 # user services
-find /userdata/system/services -type f |
+find -L /userdata/system/services -type f |
     while read SERVICE
     do
-	BASE_SERVICE=${SERVICE##*/}
-	SRVVAR=__SERVICE__${BASE_SERVICE%.*}
-	if test "${!SRVVAR}" = 1
-	then
-	    bash "${SERVICE}" "${1}" &
-	fi
+    BASE_SERVICE=${SERVICE##*/}
+    if echo "${BASE_SERVICE,,}" | grep -q -E ^[a-z].[a-z0-9_]*$; then
+        SRVVAR=__SERVICE__${BASE_SERVICE}
+        if test "${!SRVVAR}" = 1; then
+    	    bash "${SERVICE}" "${1}" &
+            echo "User Service: ${BASE_SERVICE} - service is enabled - filename [ok]"
+        else
+            echo "User Service: ${BASE_SERVICE} - service is disabled - filename [ok]"            
+        fi
+    else
+        string_new=${BASE_SERVICE//[^A-Za-z0-9_]/}
+        string_new=$(echo "$string_new" | sed 's/[[:digit:]]*//')
+        echo "User Service: $BASE_SERVICE needs rename to $string_new - filename [ko]"
+    fi
     done
 
 # system user services
-find /usr/share/batocera/services -type f |
+find -L /usr/share/batocera/services -type f |
```

Comment:
This is a behavior change not a bug fix, and should probably be in a separate PR since it's not logically related to the other changes.  Same for line 20